mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -215,10 +224,21 @@ class GBTClassificationModel private[ml](
    *
    * @param _trees  Decision trees in the ensemble.
    * @param _treeWeights  Weights for the decision trees in the ensemble.
+   * @param numFeatures  The number of features.
+   */
+  private[ml] def this(uid: String, _trees: Array[DecisionTreeRegressionModel],
```

Comment:
done, thanks, also updated the other constructor (my default intellij settings don't seem to match the suggested ones)